subsys/bluetooth/controller/ll_sw/lll_conn.h

```diff
@@ -69,11 +69,20 @@ struct lll_conn {
 		struct {
 			uint8_t initiated:1;
 			uint8_t cancelled:1;
+			uint8_t forced:1;
+		};
+
+		struct {
+			uint8_t initiated:1;
+			uint8_t cancelled:1;
+			uint8_t forced:1;
 		} central;
```

Comment:
What is the difference between these 2 structs now?

Reply:
unnamed struct is now used by a common code block and hence did not want to have a named reference as `central` or `peripheral`.

Reply:
Suggest to put that in a comment, as that is non-obvious. Looks more like a merge conflict gone wrong :P